packages/lexicons/real/fm/teal/alpha/feed/defs.json

```diff
@@ -0,0 +1,79 @@
+{
+  "lexicon": 1,
+  "id": "fm.teal.alpha.feed.defs",
+  "description": "This lexicon is in a not officially released state. It is subject to change. | Misc. items related to feeds.",
+  "defs": {
+    "playView": {
+      "type": "object",
+      "required": ["trackName", "artistName"],
+      "properties": {
+        "trackName": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 256,
+          "maxGraphemes": 2560,
+          "description": "The name of the track"
+        },
+        "trackMbId": {
+          "type": "string",
+          "description": "The Musicbrainz ID of the track"
+        },
+        "recordingMbId": {
+          "type": "string",
+          "description": "The Musicbrainz recording ID of the track"
+        },
+        "duration": {
+          "type": "integer",
+          "description": "The length of the track in seconds"
+        },
+        "artistName": {
```

Comment:
There's a property for a single `artistName` but then another property for `artistMbIds`, which is correctly an array. I think `artistName` should be `artistNames` plural and be an array as well, otherwise you couldn't put multiple artists in there structurally.

Reply:
agreed